config/test.json

```diff
@@ -9,5 +9,14 @@
     "s3": {
       "bucket": "opencollective-test"
     }
+  },
+  "cache": {
+    "homepage": {
+      "disabled": true
+    }
+  },
+  "mailgun": {
```

Comment:
Multiple tests are setting `config.mailgun.user`. These tests were failing if the config for `mailgun` was `undefined`.